### PR TITLE
use the logs as slice instead of map

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
adapt to logs and events changes -> use slice instead of map in order to maintain the order

### CODE REVIEW

**Review by iulianpascalau — COMMENTED** [review]
Please also make the required changes on the https://github.com/ElrondNetwork/elastic-indexer-go/pull/88 PR

**Comment by miiu96** [comment]
Add pr description

File: process/logsevents/logsAndEventsProcessor.go [hunk 1]
```diff
@@ -108,8 +107,8 @@ func (lep *logsAndEventsProcessor) ExtractDataFromLogs(
 			continue
 		}
 
-		events := txLog.GetLogEvents()
-		lep.processEvents(txLog.TxHash, txLog.GetAddress(), events)
+		events := txLog.LogHandler.GetLogEvents()
```
Comment (iulianpascalau) [inline process/logsevents/logsAndEventsProcessor.go:110]: Please re-check L106

File: process/logsevents/logsAndEventsProcessor.go [hunk 2]
```diff
@@ -193,7 +192,7 @@ func (lep *logsAndEventsProcessor) PrepareLogsForDB(
 			continue
 		}
 
-		logs = append(logs, lep.prepareLogsForDB(txLog.TxHash, txLog, timestamp))
+		logs = append(logs, lep.prepareLogsForDB(txLog.TxHash, txLog.LogHandler, timestamp))
```
Comment (iulianpascalau) [inline process/logsevents/logsAndEventsProcessor.go:195]: Please re-check L191

File: process/logsevents/logsAndEventsProcessor.go (hunk 1 above)
Comment (bogdan-rosianu) [reply process/logsevents/logsAndEventsProcessor.go:110]: fixed

**Review by iulianpascalau — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Pull request merged**